File: code-examples/python/categories/update.py
import http.client

conn = http.client.HTTPConnection("api.marketcloud.it")

payload = "{\"name\" : \"Wine Company\"}"

headers = {
    'authorization': "YourPublicKey:YourToken",
    'accept': "application/json",
    'content-type': "application/json"
    }

conn.request("PUT", "/v0/categories/:id", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: code-examples/csharp/categories/update.cs
var client = new RestClient("http://api.marketcloud.it/v0/categories/:id");
var request = new RestRequest(Method.PUT);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddParameter("application/json", "{\"name\" : \"Wine Company\"}", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);